Description: correct some typos in source
Author: Jörg Frings-Fürst <debian@jff-webhosting.net>
Forwarded: not-needed
Last-Update: 2014-10-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/src/xsane-text.h
===================================================================
--- trunk.orig/src/xsane-text.h
+++ trunk/src/xsane-text.h
@@ -854,7 +854,7 @@ The format of [DEVICE] is backendname:de
  -n, --no-mode-selection      disable menu for XSane mode selection\n\
 \n\
  -F, --Fixed                  fixed main window size (overwrite preferences value)\n\
- -R, --Resizeable             resizable, scrolled main window (overwrite preferences value)\n\
+ -R, --Resizable              resizable, scrolled main window (overwrite preferences value)\n\
 \n\
  -p, --print-filenames        print image filenames created by XSane\n\
  -N, --force-filename name    force filename and disable user filename selection\n\
Index: trunk/src/xsane.c
===================================================================
--- trunk.orig/src/xsane.c
+++ trunk/src/xsane.c
@@ -67,7 +67,7 @@ struct option long_options[] =
 #endif
   {"no-mode-selection", no_argument, 0, 'n'},
   {"Fixed", no_argument, 0, 'F'},
-  {"Resizeable", no_argument, 0, 'R'},
+  {"Resizable", no_argument, 0, 'R'},
   {"print-filenames", no_argument, 0, 'p'},
   {"force-filename", required_argument, 0, 'N'},
   {0, }
@@ -5787,7 +5787,7 @@ static int xsane_init(int argc, char **a
            xsane.main_window_fixed = 1;
          break;
 
-        case 'R': /* --Resizeable */
+        case 'R': /* --Resizable */
            xsane.main_window_fixed = 0;
          break;
 
